services/shotgrid_common/ayon_shotgrid_hub/update_from_ayon.py

```diff
@@ -252,6 +252,12 @@ def _create_sg_entity(
 
     logging.debug(f"Creating Shotgrid entity {sg_type} with data: {data}")
 
-    return sg_session.create(sg_type, data)
+    try:
```

Comment:
If the entity already exists, it will raise, which will be handled by the `try...except` in line `68`, we would also exiting early if we just created a new one, since we are returning the newly created one.

Reply:
got it